app/models/user.js

```diff
@@ -0,0 +1,14 @@
+import DS from 'ember-data';
+
+App.User = DS.Model.extend({
```

Comment:
global variables are long gone. you should start using `ember generate` to generate models.